README.md

```diff
@@ -8,5 +8,40 @@ All Routes requires these parameters: `session`, `semester`, `degree`, `section`
 | `/`                        | _            | Fetch the timetable from `timetable.lgu.edu.pk` and return it in **JSON format**|
 | `/gc-integration`            | _            | Create events on google calendar based on the official timetable.               |
 
+<br>
+
+### Technologies used 🛠
+
+<p align="left"> 
+
+ <img src="https://img.shields.io/badge/JavaScript-323330?style=for-the-badge&logo=javascript&logoColor=F7DF1E">
+ <img src="https://img.shields.io/badge/shell-1572B6?style=for-the-badge&logo=shell&logoColor=white">
+ 
+ <br>
+ 
+ ### Complete documents about project  📘
+ - please read [how to Setup](Docs) before diving into project.
```

Comment:
Please replace the Docs with this link https://github.com/IIvexII/LGU-TimetableAPI/blob/main/docs/Setup_Guide.md